app/src/main/java/org/p2p/wallet/auth/ui/generalerror/timer/OnboardingGeneralErrorTimerFragment.kt

```diff
@@ -17,36 +18,51 @@ import org.p2p.wallet.utils.withArgs
 import org.p2p.wallet.auth.ui.generalerror.timer.OnboardingGeneralErrorTimerContract.View as ContractView
 
 private const val ARG_TIMER_ERROR_TYPE = "ARG_TIMER_ERROR_TYPE"
+private const val ARG_PHONE_NUMBER = "ARG_PHONE_NUMBER"
+private const val ARG_COUNTRY_CODE = "ARG_COUNTRY_CODE"
 
 class OnboardingGeneralErrorTimerFragment :
     BaseMvpFragment<ContractView, Presenter>(R.layout.fragment_onboarding_general_error_timer),
     ContractView {
 
     companion object {
-        fun create(sourceScreen: GeneralErrorTimerScreenError): OnboardingGeneralErrorTimerFragment =
-            OnboardingGeneralErrorTimerFragment()
-                .withArgs(ARG_TIMER_ERROR_TYPE to sourceScreen)
+        fun create(
+            sourceScreen: GeneralErrorTimerScreenError,
+            countryCode: CountryCode?,
+            phoneNumber: String?
+        ): OnboardingGeneralErrorTimerFragment = OnboardingGeneralErrorTimerFragment()
+            .withArgs(
+                ARG_TIMER_ERROR_TYPE to sourceScreen,
+                ARG_COUNTRY_CODE to countryCode,
+                ARG_PHONE_NUMBER to phoneNumber
+            )
     }
 
-    private val binding: FragmentOnboardingGeneralErrorTimerBinding by viewBinding()
+    override val statusBarColor: Int = R.color.bg_lime
+    override val navBarColor: Int = R.color.bg_night
+    override val presenter: Presenter by inject { parametersOf(this, error) }
 
+    private val binding: FragmentOnboardingGeneralErrorTimerBinding by viewBinding()
     private val error: GeneralErrorTimerScreenError by args(ARG_TIMER_ERROR_TYPE)
 
-    override val presenter: Presenter by inject { parametersOf(this, error) }
+    private val phoneNumber by args<String?>(ARG_PHONE_NUMBER, null)
+    private val countryCode by args<CountryCode?>(ARG_COUNTRY_CODE, null)
```

Comment:
imho easier to read and less code:
```suggestion
    private val countryCode: CountryCode? by args(ARG_COUNTRY_CODE)
```

Reply:
I'm ok with that